test/unit/spec/common/errorDisplay-Spec.js

```diff
@@ -0,0 +1,232 @@
+            }
+        }
+    });
+    let cssBaseClass;
+
+    describe('The error display module', () => {
+        it('loads', () => {
+            expect(ErrorDisplay).not.toBe(null);
+        });
+
+        it('has expected functions', () => {
+            const functions = ['make', 'defaultAdvice', 'cssBaseClass'];
+            functions.forEach((f) => {
+                expect(ErrorDisplay[f]).toBeDefined();
+            });
+        });
+    });
+
+    describe('The UI module attribute defaultAdvice', () => {
+        it('should return a string about contacting support', () => {
+            expect(ErrorDisplay.defaultAdvice).toContain('kbase.us/support');
+        });
+    });
+
+    describe('The UI module attribute cssBaseClass', () => {
+        it('should return a string to use as a CSS base class', () => {
+            expect(ErrorDisplay.cssBaseClass).toContain('kb-error-display');
+        });
+    });
+
+    describe('An errorDisplay instance', () => {
+        let errorDisplayInstance;
+        beforeEach(() => {
+            errorDisplayInstance = ErrorDisplay.make({
+                model: validInput,
+            });
+        });
+
+        it('has a make function that returns an object', () => {
+            expect(errorDisplayInstance).not.toBe(null);
+        });
+
+        it('has the required methods', () => {
+            expect(errorDisplayInstance.start).toBeDefined();
+            expect(errorDisplayInstance.stop).toBeDefined();
+        });
+
+        it('should start and render an error', async () => {
+            const node = document.createElement('div');
+            cssBaseClass = ErrorDisplay.cssBaseClass;
+
+            await errorDisplayInstance.start({ node: node });
+            expect(node.classList).toContain(`${cssBaseClass}__container`);
+            expect(node.innerHTML).toContain('Error stacktrace');
+        });
+
+        it('should clean up after itself', async() => {
+            const node = document.createElement('div');
+            cssBaseClass = ErrorDisplay.cssBaseClass;
+
+            await errorDisplayInstance.start({ node: node });
+            expect(node.classList).toContain(`${cssBaseClass}__container`);
+            await errorDisplayInstance.stop();
+            expect(node.innerHTML).toBe('');
+        });
+    });
+
+    describe('The ErrorDisplay module should render different errors', () => {
+
+        const invalidArgs = [
+            null,
+            undefined,
+            {},
+            {this: 'that'},
+            Props.make({}),
+        ]
+        invalidArgs.forEach((input) => {
+            it(`should throw an error with invalid input ${ JSON.stringify(input) }`, () => {
+                try {
+                    ErrorDisplay.make(input)
+                }
+                catch (error) {
+                    expect(error.toString()).toContain('Invalid input for the ErrorDisplay module')
+                    // expect(typeof error).toBe(Error);
```

Comment:
Remove comments?

Reply:
oh crap, didn't mean to leave those in! Thanks for the 👀